src/log/levels/helpers.ts

```diff
@@ -0,0 +1,57 @@
+import {StrongInt} from '@toreda/strong-types';
+
+export function LogLevelEnable(obj: StrongInt, level: number): void {
+	if (!isPositiveInteger(level)) {
+		return;
+	}
+
+	obj(obj() | level);
+}
+
+export function LogLevelEnableMultiple(obj: StrongInt, levels: number[]): void {
```

Comment:
Minor `TypeScript` style issue: for consistency, function names start with a lowercase letter.

- [ ] Change `LogLevelEnableMultiple` to `logLevelEnableMultiple`